BPOR/BPOR.Domain/Entities/ParticipantDbContext.cs

```diff
@@ -65,7 +66,7 @@ public IQueryable<Participant> GetParticipantByLinkedIdentifiers(List<Identifier
     {
         var values = identifiers.Select(id => id.Value).ToList();
 
-        return Participants.Where(p => p.ParticipantIdentifiers.Any(pi => values.Contains(pi.Value)));
+        return Participants.Where(p => p.ParticipantIdentifiers.Any(pi => values.Contains(pi.Value))).Include(x => x.participantContactMethod);
```

Comment:
moved include to the calling function, also added a include for the participant location to avoid future issues with trying to add another entry for a participant that has already had one added.